src/libs/actions/IOU.js

```diff
@@ -2253,6 +2253,23 @@ function editMoneyRequest(transaction, transactionThreadReportID, transactionCha
     }
 }
 
+/**
+ * Updates the amount and currency fields of a money request
+ *
+ * @param {String} transactionID
+ * @param {Number} transactionThreadReportID
```

Comment:
AFAIK, all report IDs should be strings. Can you double-check the type and update the param docs for the other `updateMoneyRequest*` methods?

Reply:
Ah yes, we always pass `report.reportID` which is defined as a string [here](https://github.com/Expensify/App/blob/78dae42d3bdd7d80902085195682887726c13713/src/types/onyx/Report.ts#L80-L81). Updated all params type